apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -208,9 +220,10 @@ public override string ConnectionString
         public Task<SpannerTransaction> BeginTransactionAsync(
             CancellationToken cancellationToken = default(CancellationToken))
         {
-            return BeginTransactionImpl(cancellationToken, new TransactionOptions {
+            return BeginTransactionImplAsync(new TransactionOptions
+            {
                 ReadWrite = new TransactionOptions.Types.ReadWrite()
-            }, TransactionMode.ReadWrite);
+            }, TransactionMode.ReadWrite, cancellationToken);
```

Comment:
there are a few cases where None is by design, but those occur much less frequently